src/main/java/bio/terra/service/upgrade/Migrate.java

```diff
@@ -1,363 +1,70 @@
-        }
-
-        return row;
-    }
-
-    private void makeDeploymentTable() {
-        final String schemaCreate = "CREATE SCHEMA IF NOT EXISTS migrate";
-        final String tableCreate = "CREATE TABLE IF NOT EXISTS migrate.deployment_v1 (" +
-            " dep_version integer primary key," +
-            " id text," +
-            " locking_pod_name text)";
-
-        try (Connection connection = dataSource.getConnection();
-             PreparedStatement schemaStatement = connection.prepareStatement(schemaCreate);
-             PreparedStatement tableStatement = connection.prepareStatement(tableCreate)) {
-
-            startTransaction(connection);
-            schemaStatement.execute();
-            tableStatement.execute();
-            commitTransaction(connection);
-
-        } catch (SQLException ex) {
-            throw new MigrateException("Table create failed", ex);
-        }
-    }
-
-    private void migrateDatabase(String changesetFile, DataSource dataSource) {
-        try (Connection connection = dataSource.getConnection()) {
+    public void migrateDatabase() {
+        String changesetFile = dataRepoJdbcConfiguration.getChangesetFile();
+        try (Connection connection = dataRepoJdbcConfiguration.getDataSource().getConnection()) {
             Liquibase liquibase = new Liquibase(changesetFile,
                 new ClassLoaderResourceAccessor(),
                 new JdbcConnection(connection));
-            DatabaseChangeLogLock[] locks = liquibase.listLocks();
-            for (DatabaseChangeLogLock lock : locks) {
-                logger.info(String.format("dbChangeLogLock changeSet: %s, id: %s, lockedBy: %s, granted: %s",
-                    changesetFile, lock.getId(), lock.getLockedBy(), lock.getLockGranted()));
 
-                /*
-                 * We can get into this state where one of the APIs is running migrations and gets shut down so that
-                 * another API container can run. It will result in a lock that doesn't get released. This is similar
-                 * to the problems we will have from deploying multiple containers at once that try to run migrations.
-                 */
-                logger.warn("forcing lock release");
-                liquibase.forceReleaseLocks();
-            }
             logger.info(String.format("dropAllOnStart is set to %s", migrateConfiguration.getDropAllOnStart()));
-            if (migrateConfiguration.getDropAllOnStart()) {
+            String[] profiles = env.getActiveProfiles();
+            boolean allowDropAllOnStart = Arrays.stream(profiles).anyMatch(env -> env.contains("dev")
+                || env.contains("test") || env.contains("int"));
+            logger.info(String.format("Allow dropAllOnStart is set to %s", allowDropAllOnStart));
+
+            if (allowDropAllOnStart && migrateConfiguration.getDropAllOnStart()) {
                 liquibase.dropAll(); // drops everything in the default schema. The migrate schema should be OK
```

Comment:
We actually probably want a log message here to show that the drop actually happened